src/util/XDRCereal.cpp

```diff
@@ -37,8 +37,17 @@ cereal_override(cereal::JSONOutputArchive& ar,
 }
 
 void
-cereal_override(cereal::JSONOutputArchive& ar, const stellar::Asset& s,
+cereal_override(cereal::JSONOutputArchive& ar, const stellar::Asset& asset,
                 const char* field)
 {
-    xdr::archive(ar, stellar::assetToString(s), field);
+    xdr::archive(
+        ar,
+        std::make_tuple(
```

Comment:
This should be changed to render native asset as "native" (no sub-object).

Reply:
I only know one way to print an `xvector` of `asset` and that is to create a vector where each element is an NVP of the corresponding element of `xvector`. In other words, I'm not sure how I can render native asset without a sub-object and other assets as a sub-object. I made a change to make it clear that native asset is a native asset and there is no `assetCode` and `issuer` like following:

```
"pathPaymentStrictReceiveOp": {    
    "sendAsset": {                                
        "assetType": "native",                                
        "assetCode/issuer": "N/A"                                
    },
...
```

Let me know what you think!

Reply:
Thank you for https://github.com/jonjove/stellar-core/commit/d7472a00f05788f1165722054174d0426cba13f5! I would have never come up with this idea.

I updated the PR with the change. I also tested nested containers (which we don't have now, but for future reference & out of curiosity), and it seems to work exactly as we'd expect. https://github.com/hidenori-shinohara/xdrpp/commit/abe8047a86fa7054d3ed4469d38bc53683b06094

Reply:
This is how asset information is printed with this change:
```
                        "body": {
                            "type": "PATH_PAYMENT_STRICT_RECEIVE",
                            "pathPaymentStrictReceiveOp": {
                                "sendAsset": "NATIVE",
                                "sendMax": 1000000000,
                                "destination": "GDMFMYNJMV3KYKUYVWDNFH6IEGSL524LBQR5TXZ54BPOKGKS3DIMFIAB",
                                "destAsset": "NATIVE",
                                "destAmount": 2000000000,
                                "path": [
                                    {
                                        "assetCode": "CENTUS",
                                        "issuer": "GCD5VCGHNLC46D5O2SAZCFFP3ZF7JRBQZGSFINXWNY5HCEOE42PH3XH5"
                                    },
                                    {
                                        "assetCode": "USD",
                                        "issuer": "GDJAG52BJUFI6RBKFTNUEJBDZODJPZBLIHN3SXMR7QIXNRAOJK3XL6P3"
                                    }
                                ]
                            }
                        }
```